src/main/scala/com/typesafe/sbt/packager/windows/WindowsPlugin.scala

```diff
@@ -57,6 +57,8 @@ object WindowsPlugin extends AutoPlugin {
       "-cultures:en-us"),
     wixProductId := WixHelper.makeGUID,
     wixProductUpgradeId := WixHelper.makeGUID,
+    wixMajorVersion := 4,
```

Comment:
Can you change this to `3`, so we remain backwards compatible and the tests run.